utils/build.ps1

```diff
@@ -167,9 +171,13 @@ param
   [string] $Variant = "Asserts",
   [switch] $Clean,
   [switch] $DebugInfo,
+  [parameter(ParameterSetName = "LocalCaching")]
```

Comment:
Using parameter set names here to isolate the local options from the remote options.